lms/djangoapps/certificates/apis/v0/views.py

```diff
@@ -0,0 +1,112 @@
+""" API v0 views. """
+import logging
+
+from rest_framework.authentication import SessionAuthentication
+from rest_framework.generics import GenericAPIView
+from rest_framework.permissions import IsAuthenticated
+from rest_framework.response import Response
+from rest_framework_oauth.authentication import OAuth2Authentication
+
+from opaque_keys import InvalidKeyError
+from opaque_keys.edx.keys import CourseKey
+
+from openedx.core.lib.api import permissions
+from lms.djangoapps.certificates.api import get_certificate_for_user
+
+log = logging.getLogger(__name__)
+
+
+class CertificatesDetailView(GenericAPIView):
+    """
+        **Use Case**
+
+            * Get the details of a certificate for a specific user in a course.
+
+        **Example Request**
+
+            GET /api/certificates/v0/certificates/{username}/{course_id}
```

Comment:
ideally this interface would conform to the [guidelines for identifying resources](https://openedx.atlassian.net/wiki/display/AC/edX+REST+API+Conventions#edXRESTAPIConventions-2.IdentifyingResources), specifically the "composite resource."  In practice I think the most fitting approach from those listed is the comma-delimited style, since that overtly signifies a result containing a single resource (whereas the query parameter style could be expected to return an array of results).